docs/operator-manual/upgrading/2.14-3.0.md

```diff
@@ -181,6 +181,17 @@ spec:
         namespace: guestbook
 ```
 
+### Upgraded Helm version with breaking changes 
+Helm was upgraded to 3.17.1.   
+This may require changing your `values.yaml` files for subcharts, if the `values.yaml` contain a section with a `null` object.   
+See related issue in [Helm GitHub repository](https://github.com/helm/helm/issues/12469)   
+See Helm 3.17.1 [release notes](https://github.com/helm/helm/releases/tag/v3.17.1)   
+Example of such a [problem and resolution](https://github.com/argoproj/argo-cd/pull/22035/files)   
+Explanation:
+- Prior to Helm 3.17.1, `null` object in `values.yaml` resulted in a warning: `cannot overwrite table with non table upon performing helm template`, and the resulting K8s object was not overridden with the invalid `null` value.
```

Comment:
Would the repo server log this if someone was impacted?

Reply:
I am not sure, didn't have the chance to test yet, is it required to log it?

Reply:
No not required, I was thinking it could help users determine if they are impacted.